libs/designsystem/src/lib/components/dropdown/dropdown.component.ts

```diff
@@ -193,10 +200,11 @@ export class DropdownComponent
   constructor(
     private renderer: Renderer2,
     private elementRef: ElementRef<HTMLElement>,
-    private changeDetectorRef: ChangeDetectorRef
+    private changeDetectorRef: ChangeDetectorRef,
+    private keyboardHandlerService: KeyboardHandlerService
   ) {}
 
-  onToggle(event: Event) {
+  onToggle(event: MouseEvent) {
```

Comment:
Revisiting this I think it can use some more inspection in general. 

Try the following steps: 
1. Set `popover = false` 
2. In Safari go to `http://localhost:4200/home/showcase/dropdown`
3. Click on the dropdown saying: 'Dropdown with plain text' 
4. Select 'item 1' 
5. Notice that the dropdown button has a blue outline. 

Try to repeat these steps but set `popover = true` instead. Notice that it does not have a blue outline. 

Now do the same in Chrome and notice that it does have a blue outline. 

Using tab, arrow keys and enter for navigation In Safari however results in a blue outline. 

So... This might be worth looking more into.

Reply:
#1792